fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/base/SingleFluidStorage.java

```diff
@@ -28,7 +28,7 @@
 
 /**
  * A storage that can store a single fluid variant at any given time.
- * Implementors should at least override {@link #getCapacity(FluidVariant)},
+ * Implementors should at least override {@code getCapacity(FluidVariant)},
```

Comment:
That should exist in the superclass, and the link is quite nice here. (Unless it was a javadoc error?)

Reply:
@Technici4n Yes, this is a javadoc error. Apparently this is caused by generic argument - and I could not find a way to satisfy both the actual renderer and IDEA linting.

Reply:
Ahhh. What about `{@link #getCapacity(TransferVariant) getCapacity(FluidVariant)}` ?